### DISCUSSION TITLE
Version for systemd-less distros ?

### DISCUSSION THREAD

**Discussion by hemanzur** [AdnanHodzic/auto-cpufreq#516] [Ideas]
Great job
Is there any way of running this service in a distro without systemd ? (like mx linux)

Thanks

**Comment by AdnanHodzic** [comment]
Thanks, this has been requested as part of #29. As I stated in[ this comment,](https://github.com/AdnanHodzic/auto-cpufreq/issues/29#issuecomment-766354087) until support is there I would advise you to use Snap package.

Someone stated they made [support for it as part of separate repo](https://github.com/AdnanHodzic/auto-cpufreq/issues/29#issuecomment-1327613315), but since this way it's missing on all auto-cpufreq features from `master` branch, hence if you want to give it a try and [contribute to the project](https://github.com/AdnanHodzic/auto-cpufreq/#code-contribution) and you will be credited for your work as part of [future release](https://github.com/AdnanHodzic/auto-cpufreq/releases).
